templates/injector-psp.yaml

```diff
@@ -0,0 +1,44 @@
+{{- if and (eq (.Values.injector.enabled | toString) "true" ) (and (eq (.Values.global.enabled | toString) "true") (eq (.Values.global.psp.enable | toString) "true") ) }}
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: {{ template "vault.fullname" . }}-agent-injector
+  labels:
+    app.kubernetes.io/name: {{ include "vault.name" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+  annotations:
+    {{ tpl .Values.global.psp.annotations . | nindent 4 | trim }}
```

Comment:
Can you make these support both YAML and YAML-formatted strings like the other annotations in the chart?

Reply:
OK, I will make the changes.